### PR TITLE
feat(console): FE changes for Open Telemetry

### PR DESCRIPTION
**Pull request by illichso**
## Issue

https://gravitee.atlassian.net/browse/APIM-7312

## Description

A small description of what you did in that PR.

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-ykhiqbfmjj.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html [hunk 1]
```diff
@@ -300,6 +300,27 @@ <h3>Temporal</h3>
...
+          [matTooltip]="providedConfigurationMessage"
+          [matTooltipDisabled]="!isReadonly('logging.user.displayed')"
+          class="api-logging__form__card__form-field"
+        >
+          <mat-icon *ngIf="isReadonly('logging.user.displayed')" gioFormPrefix>lock</mat-icon>
+          <gio-form-label>Add</gio-form-label>
+          <mat-slide-toggle
+            gioFormSlideToggle
+            formControlName="displayed"
+            aria-label="Display end user on API Logging (in case of OAuth2/JWT plan)"
```
Comment (guillaumelamirand) [inline gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html]: Should we change this label for OpenTelemetry ?

File: gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html [hunk 2]
```diff
@@ -300,6 +300,27 @@ <h3>Temporal</h3>
       </mat-card-content>
     </mat-card>
 
+    <mat-card class="api-logging__form__card" formGroupName="OpenTelemetry">
+      <mat-card-content>
+        <h2 gioTableOfContents>OpenTelemetry</h2>
+        <p>Activate enabling or disabling OpenTelemetry tracing for individual APIs.</p>
+        <gio-form-slide-toggle
+          [matTooltip]="providedConfigurationMessage"
+          [matTooltipDisabled]="!isReadonly('logging.user.displayed')"
```
Comment (guillaumelamirand) [inline gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html]: Should this refer to OpenTelemetry configuration instead ?

File: gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html [hunk 3]
```diff
@@ -300,6 +300,27 @@ <h3>Temporal</h3>
...
+          class="api-logging__form__card__form-field"
+        >
+          <mat-icon *ngIf="isReadonly('logging.user.displayed')" gioFormPrefix>lock</mat-icon>
```
Comment (guillaumelamirand) [inline gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html]: Same here, `logging.user.displayed` needs to refer to OpenTelemetry settings

File: gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html [hunk 4]
```diff
@@ -300,6 +300,27 @@ <h3>Temporal</h3>
       </mat-card-content>
     </mat-card>
 
+    <mat-card class="api-logging__form__card" formGroupName="OpenTelemetry">
```
Comment (guillaumelamirand) [inline gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html]: The panel doesn't seem to do anything, nothing is saved when we change the toggle.

File: gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-message-settings/api-runtime-logs-message-settings.component.html [hunk 5]
```diff
@@ -161,6 +161,19 @@
           <mat-hint> Support EL (e.g: &#123; #message.headers['Content-Type'][0] == 'application/json' &#125;)</mat-hint>
         </mat-form-field>
       </div>
+
+      <div class="settings__form-control">
```
Comment (guillaumelamirand) [inline gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-message-settings/api-runtime-logs-message-settings.component.html:165]: Try to play with this form doesn't update my api configuration.

**Review by guillaumelamirand — CHANGES_REQUESTED** [review]

**Review by guillaumelamirand — APPROVED** [review]

**Review by phiz71 — APPROVED** [review]

**Pull request merged**